### PR TITLE
Changes at Duluth tournament

### PR DESCRIPTION
**Pull request by Speedy6451**

### CODE REVIEW

File: src/main/java/com/team2502/robot2023/RobotContainer.java [hunk 1]
```diff
@@ -64,22 +64,20 @@ private void configureButtonBindings() {
         JoystickButton RunIntakeBack = new JoystickButton(JOYSTICK_DRIVE_LEFT, Constants.OI.RUN_INTAKE_BACK);
         RunIntakeBack.whileTrue(new RunIntakeCommand(INTAKE, CONVEYOR, -0.5, -0.55, -0.4));
 
-        JoystickButton ElevatorBot = new JoystickButton(JOYSTICK_OPERATOR, Constants.OI.ELEVATOR_BOT);
-        ElevatorBot.onTrue(new SetElevatorCommand(ELEVATOR, Constants.Subsystems.Elevator.ElevatorPosition.BOTTOM))
-                .onFalse(new InstantCommand(() -> ELEVATOR.stop()));
+        new JoystickButton(JOYSTICK_OPERATOR, Constants.OI.ELEVATOR_STO)
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/RobotContainer.java:67]: Recommend adding comments to know what the button is doing

File: src/main/java/com/team2502/robot2023/RobotContainer.java [hunk 2]
```diff
@@ -64,22 +64,20 @@ private void configureButtonBindings() {
...
+            .whileTrue(new SetElevatorCommand(ELEVATOR, Constants.Subsystems.Elevator.ElevatorPosition.GROUND_PICKUP));
 
-        JoystickButton ElevatorMid = new JoystickButton(JOYSTICK_OPERATOR, Constants.OI.ELEVATOR_MID);
-        ElevatorMid.onTrue(new SetElevatorCommand(ELEVATOR, Constants.Subsystems.Elevator.ElevatorPosition.MIDDLE))
-                .onFalse(new InstantCommand(() -> ELEVATOR.stop()));
+        new JoystickButton(JOYSTICK_OPERATOR, Constants.OI.ELEVATOR_GROUND)
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/RobotContainer.java:70]: Recommend adding comments to know what the button is doing

File: src/main/java/com/team2502/robot2023/RobotContainer.java [hunk 3]
```diff
@@ -64,22 +64,20 @@ private void configureButtonBindings() {
...
+            .whileTrue(new SetElevatorCommand(ELEVATOR, Constants.Subsystems.Elevator.ElevatorPosition.GROUND_PICKUP, Constants.Subsystems.Elevator.ElevatorPitch.GROUND_PICKUP));
 
-        JoystickButton ElevatorTop = new JoystickButton(JOYSTICK_OPERATOR, Constants.OI.ELEVATOR_TOP);
-        ElevatorTop.onTrue(new SetElevatorCommand(ELEVATOR, Constants.Subsystems.Elevator.ElevatorPosition.TOP))
-                .onFalse(new InstantCommand(() -> ELEVATOR.stop()));
+        new JoystickButton(JOYSTICK_OPERATOR, Constants.OI.ELEVATOR_BOT)
+            .whileTrue(new SetElevatorCommand(ELEVATOR, Constants.Subsystems.Elevator.ElevatorPosition.CONE_BOTTOM, Constants.Subsystems.Elevator.ElevatorPitch.CONE_BOTTOM));
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/RobotContainer.java:74]: Recommend adding comments to know what the button is doing

File: src/main/java/com/team2502/robot2023/RobotContainer.java [hunk 4]
```diff
@@ -64,22 +64,20 @@ private void configureButtonBindings() {
...
 
-        new JoystickButton(JOYSTICK_OPERATOR, OI.MANIPULATOR_OUT)
-            .onTrue(new InstantCommand(() -> ELEVATOR.setPitch(Constants.Subsystems.Elevator.ElevatorPitch.OUT), ELEVATOR));
-        new JoystickButton(JOYSTICK_OPERATOR, OI.MANIPULATOR_IN)
-            .onTrue(new InstantCommand(() -> ELEVATOR.setPitch(Constants.Subsystems.Elevator.ElevatorPitch.STOWED), ELEVATOR));
+        new JoystickButton(JOYSTICK_OPERATOR, Constants.OI.ELEVATOR_MID)
+            .whileTrue(new SetElevatorCommand(ELEVATOR, Constants.Subsystems.Elevator.ElevatorPosition.MIDDLE, Constants.Subsystems.Elevator.ElevatorPitch.CONE_MID));
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/RobotContainer.java:77]: Recommend adding comments to know what the button is doing

File: src/main/java/com/team2502/robot2023/RobotContainer.java [hunk 5]
```diff
@@ -64,22 +64,20 @@ private void configureButtonBindings() {
...
+
+        new JoystickButton(JOYSTICK_OPERATOR, Constants.OI.ELEVATOR_TOP)
+            .whileTrue(new SetElevatorCommand(ELEVATOR, Constants.Subsystems.Elevator.ElevatorPosition.CONE_TOP, Constants.Subsystems.Elevator.ElevatorPitch.CONE_TOP));
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/RobotContainer.java:80]: Recommend adding comments to know what the button is doing

File: src/main/java/com/team2502/robot2023/RobotContainer.java [hunk 6]
```diff
@@ -91,34 +89,34 @@ private void configureButtonBindings() {
...
         new JoystickButton(JOYSTICK_OPERATOR, OI.MANIPULATOR_EXTEND)
-                .onTrue(new InstantCommand(() -> ELEVATOR.setPitchSpeed(0.5), ELEVATOR))
+                .onTrue(new InstantCommand(() -> ELEVATOR.setPitchSpeed(-0.5), ELEVATOR))
                 .onFalse(new InstantCommand(() -> ELEVATOR.setPitchSpeed(0.0), ELEVATOR));
         new JoystickButton(JOYSTICK_OPERATOR, OI.MANIPULATOR_RETRACT)
-                .onTrue(new InstantCommand(() -> ELEVATOR.setPitchSpeed(-0.5), ELEVATOR))
+                .onTrue(new InstantCommand(() -> ELEVATOR.setPitchSpeed(0.5), ELEVATOR))
                 .onFalse(new InstantCommand(() -> ELEVATOR.setPitchSpeed(0.0), ELEVATOR));
 
+        new JoystickButton(JOYSTICK_OPERATOR, OI.ELEVATOR_OVERRIDE)
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/RobotContainer.java:109]: Recommend adding comments to know what the button is doing

**Review by isaacmyers11 — APPROVED** [review]

**Review by Navya1001 — APPROVED** [review]

**Pull request merged**